plugins/modules/dedicated_nasha_manage_partition.py

```diff
@@ -0,0 +1,534 @@
+                                nas_service_name,
+                                nas_partition_name,
+                                quote(acl_wanted["ip"], safe=''),
+                            )
+                        )
+
+                        wait_for_tasks_to_complete(
+                            client,
+                            "nasha",
+                            nas_service_name,
+                            res["taskId"],
+                            sleep,
+                            max_retry,
+                        )
+
+                    except APIError as api_error:
+                        module.fail_json(
+                            msg="Failed to delete partition ACL: %s" % api_error
+                        )
+
+            elif acl_wanted["action"] == "update" or acl_wanted["action"] == "create":
+                # Create or Update ACL
+                if not module.check_mode:
+                    try:
+                        res = client.post(
+                            "/dedicated/nasha/{0}/partition/{1}/access".format(
+                                nas_service_name, nas_partition_name
+                            ),
+                            ip=acl_wanted["ip"],
+                            type=acl_wanted["type"],
+                        )
+                        wait_for_tasks_to_complete(
+                            client,
+                            "nasha",
+                            nas_service_name,
+                            res["taskId"],
+                            sleep,
+                            max_retry,
+                        )
+
+                    except APIError as api_error:
+                        module.fail_json(
+                            msg="Failed to set partition ACL: %s" % api_error
+                        )
+
+
+        nas_partition_acl_changed = [(item.get('ip'), item.get('action')) for item in nas_partition_acl_wanted if item.get('action') != 'unchanged']
+        if len(nas_partition_acl_changed) == 0:
+            final_message = (final_message
+                             + " No changes in ACL configuration.\n"
+                             )
+        else:
+            final_message = (
+                final_message
+                + " And setup ACL like this {0}\n".format(
+                    nas_partition_acl_changed,
+                )
+            )
+            acl_changed = True
+    else:
+        final_message = (
+            final_message
+            + " No ACL specified.\n"
+        )
+
+    if ('acl_changed' in locals() and acl_changed) or ('partition_changed' in locals() and partition_changed) or ('snapshot_changed' in locals() and snapshot_changed) :
+        all_changed = True
+    else:
+        all_changed = False
+
+    module.exit_json(
+        msg=final_message + "{0}\n".format(
+            DRY_RUN_MSG
+        ),
+        changed=all_changed,
+    )
+
+
+def main():
+    if not HAS_OVH:
```

Comment:
This is handled by `module_utils/ovh.py`

Reply:
Hello, I removed it from the module, thanks